### PR TITLE
Add Spanish translation to community.mdx

### PR DESCRIPTION
**Pull request by shogohida**
# What does this PR do?

Adds Spanish translation to community.mdx

Fixes #15947 

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@osanseviero @omarespejel @sgugger

### CODE REVIEW

File: docs/source/es/community.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+## Los cuadernos de la comunidad:
+
+| Cuaderno     |      Descripción      |      Autor      |      |
+|:----------|:-------------|:-------------|------:|
+| [Fine-tune a pre-trained Transformer to generate lyrics](https://github.com/AlekseyKorshuk/huggingartists) | Cómo generar letras al estilo de tu artista favorito ajustando un modelo GPT-2 |  [Aleksey Korshuk](https://github.com/AlekseyKorshuk) | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/AlekseyKorshuk/huggingartists/blob/master/huggingartists-demo.ipynb) |
```
Comment (shogohida) [inline docs/source/es/community.mdx:15]: Should I translate the titles of the notebooks too? Like [Fine-tune a pre-trained Transformer to generate lyrics] into [Ajustar un transformador preentrenado para generar letras]

File: docs/source/es/community.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
+# Comunidad
+
+Esta página reagrupa los recursos 🤗 Transformers desarrollado por la comunidad.
```
Comment (osanseviero) [inline docs/source/es/community.mdx:3]:
```suggestion
Esta página agrupa los recursos de 🤗 Transformers desarrollados por la comunidad.
```

File: docs/source/es/community.mdx (hunk 1 above)
Comment (osanseviero) [reply docs/source/es/community.mdx:15]: Yes, that would be amazing :)

**Review by osanseviero — COMMENTED** [review]
Thanks a lot 🔥

File: docs/source/es/community.mdx (hunk 1 above)
Comment (shogohida) [reply docs/source/es/community.mdx:15]: Sure! I will translate them too and let you know when I'm done!

Comment (shogohida) [reply docs/source/es/community.mdx]: I added translations to titles so please review again!

**Review by sgugger — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**